packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -139,19 +142,13 @@ class _DevToolsMenu extends StatelessWidget {
         ? TextDirection.rtl
         : TextDirection.ltr;
 
-    Widget devToolsButton(ScreenMetaData screen) {
-      final title = screen.title ?? screen.id;
-      String? disabledReason;
-      if (isRelease) {
-        disabledReason = 'Not available in release mode';
-      } else if (screen.requiresFlutter && !isFlutter) {
-        disabledReason = 'Only available for Flutter applications';
-      } else if (screen.requiresDebugBuild && !isDebug) {
-        disabledReason = 'Only available in debug mode';
-      } else if (screen.requiresDartVm && isWeb) {
-        disabledReason = 'Not available when running on the web';
-      }
-
+    Widget devToolsButton(
+      String title, {
+      IconData? icon,
+      String? screenId,
+      String? disabledReason,
+      bool? forceExternal,
```

Comment:
give this a default value of 'false' so that we can write `screenId != null && !forceExternal` below. I think that would make the ternary check below a bit more readable.

Reply:
Done!

Reply:
(Actually this condition went away when I extracted a widget because they pass their own `onPressed` now)